src/js/view/controls/controlbar.js

```diff
@@ -221,6 +222,7 @@ export default class Controlbar {
         SimpleTooltip(elements.rewind.element(), 'rewind', localization.rewind);
         SimpleTooltip(elements.settingsButton.element(), 'settings', localization.settings);
         const fullscreenTip = SimpleTooltip(elements.fullscreen.element(), 'fullscreen', localization.fullscreen);
+        const muteTip = SimpleTooltip(elements.volumetooltip.element(), 'volumetooltip', localization.mute);
```

Comment:
should this be called `mutetooltip` instead ? we already have a volume tooltip

Reply:
I didn’t want to make any fundamental changes to the existing components. I’ll raise this tomorrow.

Reply:
@karimMourra `volumetooltip` is actually the slider that appears above the volume icon when you hover over it. The `muteTip` the simple text tooltip that is going to appear when focused on the volume icon. The component (`view/controls/components/volumetooltip.js`) is called `volumetooltip` so I didn't want to change that. Thoughts?